pyaedt/modules/PostProcessor.py

```diff
@@ -726,6 +728,52 @@ def spectral(self, expressions=None, setup_name=None):
             return rep
         return
 
+    @pyaedt_function_handler()
+    def emi_receiver(self, expressions=None, setup_name=None):
+        """Create an EMI receiver Report object.
+
+        Parameters
+        ----------
+        expressions : str or list, optional
+            Expression List to add into the report. The expression can be any of the available formula
```

Comment:
```suggestion
            One or more expressions to add into the report. An expression can be any of the formulas that
```
If the type is string or list, I assume that the description should be "One or more", with a developer using a string to add one expression and a list to add more than one?